src/test/java/pl/edu/mimuw/collections/MultisetTest.java

```diff
@@ -2,8 +2,45 @@
 
 import org.junit.jupiter.api.Test;
 
+import java.util.Iterator;
+import java.util.TreeMap;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class MultisetTest {
 
+  @Test
+  void basicTest() {
```

Comment:
ani nazwa testu nic mi nie powie, ani to jak sie wywali 1. assert nic mi nie powie o kolejnych, należy to rozdzielać ze względu na funkcjonalności